lyft-helper.js

```diff
@@ -1,26 +1,60 @@
-/* TODO:
--
--
--
+var lyftMethods = require('lyftPrivateMethods');
```

Comment:
this doesnt work, does it? i fixed a bug earlier where you had syntax like this, but i think it needs to be `require('./lyftPrivateMethods')` right?